specification/loadtestservice/data-plane/readme.md

```diff
@@ -23,7 +23,16 @@ These are the global settings for the loadtestservice.
 
 ```yaml
 openapi-type: data-plane
-tag: package-2022-06-01-preview
+tag: package-2022-11-01
```

Comment:
```suggestion
tag: package-2022-11-01
security: AADToken
security-scopes: https://loadtest.azure-dev.com/.default
title: LoadTestingClient
```

Reply:
I would prefer that the security/security-scopes be specified in the REST API definition and skip them here.

Reply:
@mikekistler Yes, if it's defined in swagger, there is no need to add it to readme.md

Reply:
Removed as suggested